src/components/Photo/DocumentLiveCapture.js

```diff
@@ -104,8 +103,8 @@ export default class DocumentLiveCapture extends Component {
           <Camera
             facing={'environment'}
             idealCameraHeight={IDEAL_CAMERA_HEIGHT_IN_PX}
-            className={className}
             containerClassName={containerClassName}
+            className={'docLiveCaptureFrame'}
```

Comment:
Ditto here, the `<DocumentLiveCapture />` component knows too much about the `<Camera />` component's private styles